python/src/aac/plugins/gen_protobuf/gen_protobuf_impl.py

```diff
@@ -259,11 +260,14 @@ def _get_schema_properties(interface_structures: dict[str, Definition], data_def
         sanitized_proto_field_type = remove_list_type_indicator(proto_field_type)
         description_as_proto_comment = _convert_description_to_protobuf_comment(proto_field_description, 1)
 
+        formatted_proto_field_name = proto_field_name.replace(" ", "")
+        formatted_proto_field_type = sanitized_proto_field_type.replace(" ", "")
```

Comment:
You'll probably want to move these functions into the function `_to_template_properties_dict` since that function does expect unaltered message names/types for some lookups it performs.

Reply:
Gotcha, I was looking into this now actually as this is what I am suspecting the issue is with the `enum_name` value and the disconnect